src/explore-education-statistics-common/src/modules/release/components/NationalStatisticsSection.tsx

```diff
@@ -34,17 +39,24 @@ export default function NationalStatisticsSection({
         <li>well explained and readily accessible</li>
       </ul>
       <p>
-        Once designated as National Statistics it's a statutory requirement for
-        statistics to follow and comply with the Code of Practice for Statistics
-        to be observed.
+        Our statistical practice is regulated by the Office for Statistics
+        Regulation (OSR).
+      </p>
+      <p>
+        OSR sets the standards of trustworthiness, quality and value in the{' '}
+        <a href="https://code.statisticsauthority.gov.uk/the-code/">
+          Code of Practice for Statistics
+        </a>{' '}
+        that all producers of official statistics should adhere to.
       </p>
       <p>
-        Find out more about the standards we follow to produce these statistics
-        through our{' '}
-        <a href="https://www.gov.uk/government/publications/standards-for-official-statistics-published-by-the-department-for-education">
-          Standards for official statistics published by DfE
+        You are welcome to contact us directly with any comments about how we
+        meet these standards. Alternatively, you can contact OSR by emailing{' '}
+        <a href="mailto:regulation@statistics.gov.uk">
+          regulation@statistics.gov.uk
         </a>{' '}
-        guidance.
+        or via the{' '}
+        <a href="https://osr.statisticsauthority.gov.uk/">OSR website</a>.
```

Comment:
the text Cam sent me didn't have a link here. But I presume this is fine - better if anything :)

Reply:
Yeah I thought it was better to be linked and noticed that it was on the Official Statistics version so included it.

Reply:
cool :)